### PR TITLE
Feature/ab#535 donations ux

### PR DESCRIPTION
**Pull request by oonosan**

### CODE REVIEW

File: src/DonateTo.WebApplication/ClientApp/src/app/admin/organization/organization-form/organization-step-address/address-list/address-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  @ViewChild('modalContent') public modalContent: TemplateRef<any>;
+  tplModal?: NzModalRef;
+
+  constructor(
+    private fb: FormBuilder,
+    public organizationSandbox: OrganizationSandbox,
+    private modal: NzModalService
+  ) {}
+
+  ngOnInit(): void {}
```
Comment (Lordapa) [inline src/DonateTo.WebApplication/ClientApp/src/app/admin/organization/organization-form/organization-step-address/address-list/address-list.component.ts:64]: if this is not used, remove implementation

File: src/DonateTo.WebApplication/ClientApp/src/app/admin/organization/organization-form/organization-step-address/address-list/address-list.component.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
+<form [formGroup]="addressItemFormGroup" class="address-table">
+  <nz-form-item>
+    <nz-form-control nzErrorTip="{{ 'Admin.Questions.Form.PriorityQuestionRequiredError' | translate }}">
+      <div [(ngModel)]="addresses" ngDefaultControl formControlName="itemsFormControl">
+        <nz-table
```
Comment (guillemurano) [inline src/DonateTo.WebApplication/ClientApp/src/app/admin/organization/organization-form/organization-step-address/address-list/address-list.component.html:5]: Should't we use a filterable paged table as we use in other tables?

File: src/DonateTo.WebApplication/ClientApp/src/app/admin/organization/organization-form/organization-step-contact/organization-step-contact.component.css [hunk 3]
```diff
@@ -3,3 +3,19 @@ form {
...
+}
+
+.contact-column-information {
+  margin-left: 1em;
+  font-size: 0.8em;
+  color: gray;
+}
+
+.contact-for-branch-form {
+  width: 70%;
```
Comment (guillemurano) [inline src/DonateTo.WebApplication/ClientApp/src/app/admin/organization/organization-form/organization-step-contact/organization-step-contact.component.css:19]: favor using em

File: src/DonateTo.WebApplication/ClientApp/src/app/admin/organization/organization-form/organization-step-general-information/organization-step-general-information.component.ts [hunk 4]
```diff
@@ -26,17 +26,25 @@ export class OrganizationStepGeneralInformationComponent implements OnInit {
...
+    }
 
     this.generalInformationStepForm.valueChanges.subscribe(() =>
       this.isFormValid.emit({
         value: this.isValidForm(),
         generalInformationFormModel: this.getOrganizationFormModel(),
       })
     );
+
+    if (!this.isEditOrganization) {
```
Comment (guillemurano) [inline src/DonateTo.WebApplication/ClientApp/src/app/admin/organization/organization-form/organization-step-general-information/organization-step-general-information.component.ts:43]: Delete this empty if

**Pull request merged**